index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./css/styles.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/"
+        crossorigin="anonymous">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="main-content">
+        <div class="topbar">
+            <button type="button" class="topbar-button"><i class="fab fa-apple"></i></button>
+            <button type="button" class="topbar-button">Store</button>
+            <button type="button" class="topbar-button">Mac</button>
+            <button type="button" class="topbar-button">iPad</button>
+            <button type="button" class="topbar-button">iPhone</button>
+            <button type="button" class="topbar-button">iTunes</button>
+            <button type="button" class="topbar-button">Support</button>
+            <form action="#" class="topbar-search">
+                <input type="search" placeholder="Search">
+            </form>
+        </div>
+        <div class="header">
+            <h1>What will your verse be?</h1>
```

Comment:
This header is different from the rest of the text in the original page. Just changing the tag to `h1` doesn't seem enough